src/app/facilities-and-resources/page.tsx

```diff
@@ -1,11 +1,221 @@
+                      </span>
+                      <span
+                        className="text-xl flex justify-center items-center"
+                      >
+                        {text}
+                      </span>
+                    </li>
+                  ))
+                }
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section className="relative min-h-screen h-full flex items-center justify-center">
+        <SectionDivider />
+        <div className="flex flex-col flex-wrap gap-[11px] lg:grid grid-cols-12 grid-rows-12 py-16 2xl:px-16">
+          <div className="order-2 w-full h-auto border border-white p-[14px] col-start-1 col-span-3 row-start-1 row-span-7 lg:order-1 ">
+            <h2 className={`${alumniSans.className} font-bold bg-transparent text-4xl text-darkBlue ${styles.boxshadow}`}>
+              01
+            </h2>
+            <h3 className={`${alumniSans.className} font-bold text-5xl ${styles.textstroke}`}>
+              Athena Data Server
+            </h3>
+            <p>
+              Athena is an internal data server for students who are doing their
+              BTech in Computer Science and Engineering at the National
+              Institute of Technology, Calicut. All the students have an account
+              in Athena DataServer from their second year onwards. All users are
+              assigned a fixed amount of disk quota. The users can also host
+              their personal website.
+            </p>
+            <div className="py-[14px]">
+              <Link
+                href="https://athena.nitc.ac.in/how-to-athena/"
+                label="How to use Athena"
+              />
+            </div>
+          </div>
+          <div className="order-3 w-full h-full border border-white p-[14px] col-start-4 col-span-5 row-start-1 row-span-7 border border-dashed border-white p-[14px]">
+            <h2 className={`${alumniSans.className} font-bold text-4xl text-darkBlue ${styles.boxshadow}`}>
+              02
+            </h2>
+            <h3 className={`${alumniSans.className} font-bold text-5xl ${styles.textstroke}`}>
+              CSED Web Server
+            </h3>
+            <p>
+              CSED Web Server is official website hosting server of department.
+            </p>
+            <div className="py-3.5 flex flex-col gap-1">
+              {
+                csedServers.map(({ name, link }, index) => (
+                  <span key={index} className="">
+                    <Link href={link} label={name} openInNewTab />
+                  </span>
+                ))
+              }
+            </div>
+          </div>
+          <div className="w-full h-full order-1 col-start-9 col-span-4 row-start-1 row-span-7 lg:border border-white lg:order-3 relative flex items-center lg:justify-center">
+            <div className="hidden lg:block absolute top-0 right-0">
```

Comment:
`<div className="select-none hidden lg:block absolute top-0 right-0">`

Reply:
This is resolved.